pkg/apis/flow/v1alpha1/synchronizer_types.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2021 TriggerMesh Inc.
```

Comment:
Shouldn't this be 2022?

Reply:
Updating.
The reason all these files had 2021 in the header is that they were actually created in 2021 while the review is taking place in 2022.